source/databases-collections.txt

```diff
@@ -0,0 +1,272 @@
+   :language: scala
+   :copyable:
+   :dedent:
+
+.. warning:: Dropping a Collection Deletes All Data in the Collection
+
+   Dropping a collection from your database permanently deletes all
+   documents and all indexes within that collection.
+
+   Drop a collection only if the data in it is no longer needed.
+
+Configure Read and Write Operations
+-----------------------------------
+
+You can control how the driver routes read operations by setting a **read preference**.
+You can also control options for how the driver waits for acknowledgment of
+read and write operations on a replica set by setting a **read concern** and a
+**write concern**.
+
+By default, databases inherit these settings from the ``MongoClient`` instance,
+and collections inherit them from the database. However, you can change these
+settings on your database by using the ``withReadPreference()`` method.
+
+The following example accesses a database while specifying the read preference of the
+database as ``secondary``:
+
+.. literalinclude:: /includes/databases-collections/databases-collections.scala
+   :start-after: start-database-read-prefs
+   :end-before: end-database-read-prefs
+   :language: scala
+   :copyable:
+   :dedent:
+
+You can also change the read and write settings on your collections by using the ``withReadPreference()``
+method. The following example shows how to access a collection while specifying the read
+preference of the collection as ``secondary``:
+
+.. literalinclude:: /includes/databases-collections/databases-collections.scala
+   :start-after: start-collection-read-prefs
+   :end-before: end-collection-read-prefs
+   :language: scala
+   :copyable:
+   :dedent:
+
+.. tip::
+
+   To see the types of available read preferences, see the
+   `API documentation <{+api+}/org/mongodb/scala/ReadPreference$.html>`__. 
+
+To learn more about the read and write settings, see the following guides in the
+{+mdb-server+} manual:
+
+- :manual:`Read Preference </core/read-preference/>`
+- :manual:`Read Concern </reference/read-concern/>`
+- :manual:`Write Concern </reference/write-concern/>`
+
+Tag Sets
+~~~~~~~~
+
+In the {+mdb-server+}, you can apply key-value :manual:`tags
+</core/read-preference-tags/>` to replica-set
+members according to any criteria you choose. You can then use
+those tags to target one or more members for a read operation.
+
+By default, the {+driver-short+} ignores tags when choosing a member to read from. To
+instruct the {+driver-short+} to prefer certain tags, pass a ``TagSet`` instance to the
+``ReadPreference`` constructor, then pass the ``ReadPreference`` instance to the
+``MongoClientSettings`` you use to instantiate a ``MongoClient``.
+
+In the following code example, the tag set passed to the ``ReadPreference``
+constructor instructs the {+driver-short+} to prefer reads from the
+New York data center (``'dc': 'ny'``) and to fall back to the San Francisco data
+center (``'dc': 'sf'``):
```

Comment:
[s] Since `dc` is capitalized in the code example, can capitalize here for consistency:
```suggestion
New York data center (``'DC': 'ny'``) and to fall back to the San Francisco data
center (``'DC': 'sf'``):
```

Reply:
Corrected the code sample instead.